src/pages/research/mitiq.mdx

```diff
@@ -14,7 +14,7 @@ import { PageIntro, TwoColumn, Column, ReducedWidth } from '~/components/Ui';
 
 Designed with portability in mind, `mitiq` is made to slot in to your existing quantum compute workflows with ease, independent of the choice of quantum SDK.
 
-Check out the [5 year impact report](../posts/2024_mitiq_impact) for more information about the impact `mitiq` has made on the ecosystem.
+Check out the [5 year impact report](https://unitary.foundation/posts/2024_mitiq_impact) for more information about the impact `mitiq` has made on the ecosystem.
```

Comment:
```suggestion
Check out the [5 year impact report](/posts/2024_mitiq_impact) for more information about the impact `mitiq` has made on the ecosystem.
```

Reply:
In general, it's better to have relative links, so they work in all environments, including the local dev environment.
In fact, we need to change this in other places through the codebase where we use absolute links.